server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -51,6 +53,7 @@ import (
 	requestcontext "github.com/buildbuddy-io/buildbuddy/server/util/request_context"
 	gcodes "google.golang.org/grpc/codes"
 	gstatus "google.golang.org/grpc/status"
+	proto "google.golang.org/protobuf/proto"
```

Comment:
I don't think you need the name alias here, you can just remove it and move this import line up to the stanza before the aliased imports

Reply:
Fixed